cmd/chunk/main.go

```diff
@@ -67,6 +67,7 @@ var (
 	chunkSize            int64
 	waitBetweenRetries   time.Duration
 	restartDownloads     bool
+	progressDir          string
```

Comment:
Don't you need to set `progressDir` at Downloader (i.e., line 50)?

Reply:
Ooops. Sure thing! Done in 3c17cdf